ui/static/ui/js/listing.js

```diff
@@ -88,6 +88,7 @@ define('listing',
         LearningResources.loader(repoSlug, learningResourceId, $("#tab-1")[0]);
         $('.cd-panel').addClass('is-visible');
         StaticAssets.loader(repoSlug, learningResourceId, $("#tab-3")[0]);
+        $('body').css('overflow', 'hidden');
```

Comment:
To avoid the scroll overlapping between panel and background. This overlapping case failure in scroll up animation. if you scroll down and scroll background as well then press save it will not scroll up with animation.